client/src/components/App/AppToolbar/AppToolbarStyles.ts

```diff
@@ -0,0 +1,10 @@
+import { makeStyles } from '@material-ui/styles'
+
+const useStyles = makeStyles({
+    appToolbar: {
+        color: 'white',
+        backgroundColor: '#056989',
```

Comment:
Why not using the pallete colors on the theme?

Reply:
הצבעים בפלטה לא מספיק דומים, זה הצבע המדויק באפיון

Reply:
אז צריך לשנות את הצבעים בפלטה שיהיו תואמים,לא? אחרת אין לנו שימוש לפלטת צבעים

Reply:
מסכים תוקן